src/domain/Car.js

```diff
@@ -0,0 +1,21 @@
+class Car {
+  static NAME_LENGTH = 5;
+
+  name;
+  position = 0;
+
+  move(randomValue) {
+    if (randomValue >= 4) {
+      this.position += 1;
+    }
+  }
```

Comment:
randomValue 라는걸 자동차가 가지고 있을 필요가 있을까요?

```suggestion
  move(condition = true) {
    if (condition) {
      this.position += 1;
    }
  }
```

이런식으로만 해도 randomValue 에 대한 의존이 없어질 수 있을 것 같아요!